Packs/rasterize/Integrations/rasterize/rasterize.py

```diff
@@ -357,26 +462,37 @@ def rasterize_command():
 
     if not (url.startswith('http')):
         url = f'http://{url}'
-    file_name = f'{file_name}.{"pdf" if r_type == "pdf" else "png"}'  # type: ignore
+    file_name = f'{file_name}.{"pdf" if r_type == RasterizeType.PDF else "png"}'  # type: ignore
 
-    output = rasterize(path=url, r_type=r_type, width=w, height=h, wait_time=wait_time,
-                       max_page_load_time=page_load, full_screen=full_screen)
-    if r_type == 'json':
-        return_results(CommandResults(raw_response=output, readable_output="Successfully load image for url: " + url))
+    output = rasterize(path=url, r_type=r_type, width=w, height=h, wait_time=wait_time, max_page_load_time=page_load,
+                       full_screen=full_screen, r_mode=r_mode)
+    if r_type == RasterizeType.JSON:
+        return_results(CommandResults(raw_response=output, readable_output="Successfully rasterize url: " + url))
         return
 
     res = fileResult(filename=file_name, data=output)
-    if r_type == 'png':
+    if r_type == RasterizeType.PNG:
         res['Type'] = entryTypes['image']
 
     demisto.results(res)
 
 
+def get_common_args(args: dict):
+    """
+    Get commomn args.
+    :param args: dict to get args from
+    :return: width, height, rasterize mode
+    """
+    w = int(args.get('width', DEFAULT_W).rstrip('px'))
+    h = int(args.get('height', DEFAULT_H).rstrip('px'))
```

Comment:
An exception will be thrown here if the user sets the height/width to a non-numeric value. Are we okay with that? 
Why didn't you use the arg_to_number() func? because of Mypy?

Reply:
It's ok we need an int as later on we treat this as an int. arg_to_number is not fitting because of the optional `px` suffix.

Reply:
As long as you are okay with the default err msg that the int method will throw here we are good :)